collect-install.rst

```diff
@@ -46,53 +40,3 @@ You can also download from the web and install manually.
 
   Developers can also `install ODK Collect on an Android emulator <https://github.com/opendatakit/opendatakit/wiki/DevEnv-Setup>`_. However, this can be slow and buggy and is not recommended.
```

Comment:
I think there's now a page on other emulator options. It's still a bit slow but not actually buggy anymore.

Reply:
I updated the language here, but- the dev wiki still says it's buggy, and I can't find another page about emulators.